pandas/core/frame.py

```diff
@@ -1615,15 +1631,15 @@ def to_dict(self, orient="dict", into=dict):
 
     def to_gbq(
         self,
-        destination_table,
-        project_id=None,
-        chunksize=None,
-        reauth=False,
-        if_exists="fail",
-        auth_local_webserver=False,
-        table_schema=None,
-        location=None,
-        progress_bar=True,
+        destination_table: str,
```

Comment:
do these match the typings in the implementation function?

Reply:
Yes

In fact can do more or less copy paste for the whole signature from the implementation here and in `to_html`. I might separate that out